ComposeTesting/sample/src/androidTest/java/com/microsoft/device/dualscreen/composetesting/SampleTest.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ *
```

Comment:
Would it be possible to rename this to "FoldingFeatureTest" or "FoldingFeatureHelperTest"?